providers/printers.rb

```diff
@@ -47,74 +103,36 @@
           ppd_uri = printer.ppd_uri
         end
 
-
         if ppd_uri != '' and ppd != ''
```

Comment:
Presently, atribute "ppd" is not provided by the form associated with this policy, so always will be ppd="" (line 95), and condition ppd != '' will never be true, and external PPD pointed by ppd_uri will never be used.